### PR TITLE
feat: support for trusting private CA

### LINKED ISSUE #138: Support self-signed certificates
PostgreSQL Monitor fails when connecting to target with self-signed certificate

There's no option as of now in https://github.com/louislam/uptime-kuma to ignore TLS/SSL errors in PostgreSQL Monitor

However, uptime-kuma supports trusting additional certificates specified in a single file with `NODE_EXTRA_CA_CERTS`

Suggestion is to support such use case as described in https://github.com/louislam/uptime-kuma/issues/1380#issuecomment-1109481258

See also
- https://github.com/louislam/uptime-kuma/wiki/Environment-Variables
- https://nodejs.org/api/cli.html#node_extra_ca_certsfile

### PR DESCRIPTION
**Pull request by ricardoapl**
Resolves #138

### CODE REVIEW

**Comment by k3rnelpan1c-dev** [comment]
hmm, I am never sure if I like this approach or building a minimal custom image with the CA included.
There are advantages and disadvantages for both, but I will look into your PR in detail as soon as I can.

File: uptime-kuma/readme.adoc [hunk 1]
```diff
@@ -252,3 +252,16 @@ CAUTION: You need to deploy the chart first and create a token in the uptime-kum
...
+|===
+|Value |Default |Description
+
+|`extraCertificates.enabled`
+|`false`
+|Specifies whether the default certificates should be extended with extra certificates.
+
+|`extraCertificates.cacerts`
+|`""`
+|One or more certificates in PEM format.
```
Comment (k3rnelpan1c-dev) [inline uptime-kuma/readme.adoc:267]:
```suggestion
|One or more certificates in PEM format.
|===

```
Just to close the table form :wink:

**Review by k3rnelpan1c-dev — APPROVED** [review]
overall, looks good to me.

Since Node, unlike Java, can directly read extra CAs directly without the need of updating a trust-store, I see nothing wrong with doing this.

> reasoning for the comment
> I have wasted too many hours at work with our self signed CA and the ppl behind it that I am very cautious when it comes to this "feature"
> that and quite a few other "official" charts with Java backed services use init container ... which is a wast of time and resources when you scale up and down frequently.
> Thereby I generally advocate for building a custom image with the proper trust in cases where you need to change/patch anything to the likes of a trust-store, that not being the case here, I approve of this method.

Thank you for your contribution and sorry for the odd review comment.

**Comment by ricardoapl** [comment]
Thank you for taking the time to look into this pull request, I really appreciate it

**Pull request merged**